reference/lensflare.md

```diff
@@ -0,0 +1,30 @@
+## LensFlare
+
+```
+LensFlare {
+  SFBool  occlusion              FALSE
+  SFFloat transparency           0
+  MFVec2f haloSizesAndPositions  [0.17 0.5, 0.08 0.25, 0.33 0.12]
+  MFVec2f burstSizesAndPositions [0.33 0.5, 0.16 0.33, 0.5 0.1]
+}
+```
+
+### Description
+
+%figure "Lens flare effect"
+
+![lens_flare.png](images/lens_flare.png)
+
+%end
+
+The [LensFlare](#lensflare) node can be added in the `lensFlare` field of any [Camera](camera.md) and [Viewpoint](viewpoint.md) to simulate lens flare. Lens flare effect is due to the light been scattered in lens systems through generally unwanted image formation mechanisms, such as internal reflection and scattering from material inhomogeneities in the lens. To simulate lens flare a variable number of images called `halos` and `bursts` are added along a line from the light position to a point in front of the [Camera](camera.md) or [Viewpoint](viewpoint.md).
+
+### Field Summary
+
+- The `occlusion` field specifies whether or not obstacles between the [Camera](camera.md) or [Viewpoint](viewpoint.md) and the light should remove the lens flare effect. By default, the `occlusion` field is disabled because the occlusion detection is computationally expensive and should be avoided whenever possible.
```

Comment:
remove &rarr; disable
disabled &rarr; set to FALSE
avoided whenever possible. &rarr; avoided when not needed.